biome.json

```diff
@@ -3,25 +3,25 @@
   "files": {
     "maxSize": 5242880,
     "ignore": [
-      "node_modules",
-      "bower_components",
-      "dist",
-      "bin",
-      "docs",
-      ".temp",
-      "__testfixtures__",
-      "__fixtures__",
+      "node_modules/",
+      "bower_components/",
+      "dist/",
+      "bin/",
+      "docs/",
```

Comment:
Was trying to figure out why the `search-docs.ts` wasn't getting formatted. Apparently, Biome was just treating this as a string to match in any file path.